src/components/ToolMenu/index.less

```diff
@@ -52,7 +52,6 @@
       .ant-collapse-content-box {
         padding: 10px;
         overflow-y: auto;
-        max-height: 25vh;
```

Comment:
That makes the whole collapse scrollbar. The intention was to make the certain panels scrollable instead.